### PR TITLE
Add ChainStack to community RPCs.

### PR DESCRIPTION
**Pull request by cgcardona**

### CODE REVIEW

File: docs/apis/avalanchego/public-api-server.md [hunk 1]
```diff
@@ -206,3 +206,44 @@ P-Chain, C-Chain, and Index API.
...
+
+### ChainStack
+
+[ChainStack](https://chainstack.com/build-better-with-avalanche/) supports the
+C-Chain, X-Chain, P-Chain, and the Fuji Testnet.
+
+Features:
+
+- Globally distributed infrastructure for optimal performance.
+- crypto payments natively.
```
Comment (yulin-dong) [inline docs/apis/avalanchego/public-api-server.md:218]:
```suggestion
- Crypto payments natively.
```

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: docs/apis/avalanchego/public-api-server.md [hunk 2]
```diff
@@ -206,3 +206,44 @@ P-Chain, C-Chain, and Index API.
...
+- Crypto payments natively.
+- 24/7 customer support.
+
+#### Mainnet
+
+##### HTTP
+
+- For C-Chain API, the URL is `https://nd-123-145-789.p2pify.com/API_KEY/ext/bc/C/rpc`
+- For X-Chain API, the URL is `https://nd-123-145-789.p2pify.com/API_KEY/ext/bc/X/`
+- For P-Chain API, the URL is `https://nd-123-145-789.p2pify.com/API_KEY/ext/P`
```
Comment (yulin-dong) [inline docs/apis/avalanchego/public-api-server.md:227]: Can you confirm if the trailing / is needed? I assume the URLs should have the same pattern. Same with the Testnet below.

Comment (cgcardona) [reply docs/apis/avalanchego/public-api-server.md:227]: Sure let me check

Comment (soos3d) [reply docs/apis/avalanchego/public-api-server.md:227]: The trailing / in `https://nd-123-145-789.p2pify.com/API_KEY/ext/bc/X/` shouldn't be there. Sorry, my bad!

Comment (cgcardona) [reply docs/apis/avalanchego/public-api-server.md:227]: NP. TY!

**Review by yulin-dong — APPROVED** [review]

**Pull request merged**